Products/DataCollector/ApplyDataMap/incrementalupdate.py

```diff
@@ -123,6 +128,20 @@ def parent(self):
 
         return self._parent
 
+    @property
+    def relname(self):
+        try:
+            if self.__relname is NOTSET:
+                self.__relname = self.path.split('/')[-2]
+        except Exception:
+            self.__relname = None
```

Comment:
Any reason why this can't be determined in `_process_objectmap`?
If the value of `path` causes an exception, relname becomes `None`.  For error reporting, the message will reference `NoneType` rather than `NOTSET`'s type.

Reply:
That is the intention.  it should only be NOTSET before the first time it is evaluated, after being evaluated it should be either a string or None